doris/benchmark.sh

```diff
@@ -22,34 +22,35 @@ if [[ "$url" == "http"* ]]; then
 fi
 dir_name="${file_name/.tar.gz/}"
 
-# Try to stop SelectDB and remove it first if execute this script multiple times
+# Try to stop Doris and remove it first if execute this script multiple times
 set +e
-"$dir_name"/fe/bin/stop_fe.sh
-"$dir_name"/be/bin/stop_be.sh
+"$dir_name"/apache-doris-2.1.7-rc01-bin-x64/fe/bin/stop_fe.sh
+"$dir_name"/apache-doris-2.1.7-rc01-bin-x64/be/bin/stop_be.sh
 rm -rf "$dir_name"
 set -e
 
 # Uncompress
 mkdir "$dir_name"
 tar zxf "$file_name" -C "$dir_name"
-DORIS_HOME="$ROOT/$dir_name/"
+DORIS_HOME="$ROOT/$dir_name/apache-doris-2.1.7-rc01-bin-x64"
 export DORIS_HOME
 
 # Install dependencies
-sudo yum install -y mysql java-11-amazon-corretto.x86_64
-export JAVA_HOME="/usr/lib/jvm/java-11-amazon-corretto.x86_64/"
+sudo apt update
+sudo apt install -y openjdk-17-jdk
+sudo apt install -y mysql-client
+export JAVA_HOME="/usr/lib/jvm/java-17-openjdk-amd64/"
 export PATH=$JAVA_HOME/bin:$PATH
 
-IPADDR=$(hostname -i)
+sudo systemctl disable unattended-upgrades
+sudo systemctl stop unattended-upgrades
+sudo systemctl stop mssql-server
```

Comment:
Sure. In fact, I have no sense why there are some daemon processes (SQL Server and unattended upgrading) in AWS machine bought by our SRE team. In order to prevent interference with the results, I have to stop all of those.